src/pcre2_compile.c

```diff
@@ -4342,9 +4348,20 @@ while (ptr < ptrend)
 #endif
           break;  /* End \P and \p */
 
-          default:    /* All others are not allowed in a class */
+          /* All others are not allowed in a class */
+
+          default:
+          PCRE2_DEBUG_UNREACHABLE();
```

Comment:
Not allowed or not possible?

Reply:
Not possible (as far as I can tell). I have gone through `check_escape()` and made a list of every ESC_ code it can return (when passed `inclass=TRUE`), and added a case statement and a test for all of those. There should be no other escapes allowed, so I added a catch-all default which only triggers in Debug builds.